cluster/manifests/z-kubenurse/rbac.yaml

```diff
@@ -0,0 +1,64 @@
+{{ if eq .Cluster.ConfigItems.network_monitoring_enabled "true" }}
+{{ if eq .Cluster.ConfigItems.network_monitoring_check_unschedulable_nodes "false" }}
```

Comment:
Is that `if` correct or should it be `true`?

Reply:
We only need this role in case we do *NOT* want to check unschedulable nodes. then we will create node watchers to know what are the unschedulable nodes to exclude them

if we want to check all nodes (flag == true) we then do not need this roles